Slash/DB/MySQL/MySQL.pm

```diff
@@ -6295,7 +6297,7 @@ sub getCommentTextCached {
 	$opt ||= {};
 
 	my $possible_chop  = !$opt->{full} && !($opt->{mode} && $opt->{mode} eq 'archive');
-	my $abbreviate_ok  = $opt->{discussion2} && $possible_chop;
+	my $abbreviate_ok  = 0;
```

Comment:
No, this was strictly for d2 stuff. The "hey, there's more to this comment" stuff for d1 is logic of the other variety. We should look into it though.